### ISSUE TITLE
Timeline improvements

### ISSUE THREAD

**Issue by arj03** [arj03/ssb-browser-demo#32]
So loose ideas:
 - Grouping of people
 - [Popular posts](https://github.com/fraction/oasis/blob/71926c7bd53a2c031a9c3d1af47d4bb0aa94629e/src/index.js#L441)

**Comment by KyleMaas** [comment]
How would grouping of people work?

**Comment by arj03** [comment]
Maybe something like one of these:
 - Write lists in local storage
 - Write lists as private messages to yourself. Not here there is a limit here in message size, so maybe as a blob if the lists gets large enough

**Comment by arj03** [comment]
A much easier thing than the group might be to have a direct follow filter like the threads filter.

**Comment by KyleMaas** [comment]
That much, I could accomplish.  See #50.

**Comment by KyleMaas** [comment]
From here:

https://github.com/fraction/oasis/blob/d6737020110655b1b2a0c0903ea2746d02dc1966/src/models.js#L1294

It looks like they're determining popularity by quantity (and value, positive or negative) of votes within a recent time period?

**Comment by arj03** [comment]
@KyleMaas yes. I did an experiment once where the user would be able to fiddle with the variables: how much weight is votes / posts from friends compared to others. I think for now it would be best with a simple algorithm like the one in oasis :)
